src/assets/less/night.less

```diff
@@ -38,7 +38,28 @@
     color: rgb(171, 171, 171);
   }
 
-  .router-link-exact-active.router-link-active, .router-link-active {
+  .router-link-exact-active.router-link-active,
+  .router-link-active {
     color: rgb(0, 188, 140);
   }
+
+  .night:before {
+    color: rgba(51, 211, 255, 0.8);
+    content: '🌙 ';
+  }
+
+  .cold:before {
+    color: rgba(51, 211, 255, 0.8);
+    content: '\2744 ';
+  }
+
+  .day:before {
+    color: rgba(249, 168, 6, 0.8);
+    content: '☀ ';
+  }
+
+  .warm:before {
+    color: rgba(249, 168, 6, 0.8);
+    content: '☀ ';
+  }
```

Comment:
the content bits would probably be fine to stay in common or the main component file, i was just worried about styling trying to override component styles for themes